csharp/ExcelAddInInstaller/CustomActions/README.md

```diff
@@ -0,0 +1,121 @@
+# Background
+
+The purpose of this library is to add a "Custom Action" to our
+Advanced Installer package. This custom action does the actions
+needed to manipulate the Windows Registry in order to do things like
+
+1. detect whether the version of Office installed is 32 or 64 bit
+2. Add the special keys that tell Excel to open an Excel Add-In at startup
+
+# Information about this library
+
+This library is a .NET 4.8.0 Class Library with some special boilerplate
+code provided by Advanced Installer.
+
+.NET 4.8.0 is pretty old at this point, but I chose it because (I believe)
```

Comment:
I suggest to replace "I chose" with "it was chosen", and removing "(I believe)".  It you think is important to keep it as is, I would then replace "I chose" with "I (Corey) chose".  My reasoning for the suggestions: giving a rationale for why something was chosen makes sense.  Any statement is assumed to come from some set of "I believe" so that's redundant unless you want to be explicitly apologetic, and I don't think this is the time or place to be (because IMHO that part doesn't add value in the context of somebody reading this text with a need for context towards some concrete goal).

Reply:
Fixed, thank you.